notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -202,7 +202,6 @@
       "outputs": [],
```

Comment:
<p>Why not writing the commands that give these permission in case of any error? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I was conforming to existing syntax I found in one of Andrew's notebook.</p>